### ISSUE TITLE
Markdown Link wrongly rendered

### ISSUE THREAD

**Issue by obstschale** [TandoorRecipes/recipes#1919]
### Tandoor Version

1.2.7

### Setup

Docker / Docker-Compose

### Reverse Proxy

Traefik

### Bug description

I added a markdown link to a step. But the link is wrongly rendered. Only the "text" part is linked not the full domain path.

Markdown editor in edit mode
![CleanShot 2022-07-09 at 08 15 38](https://user-images.githubusercontent.com/1058710/178094246-ce32ca4a-e281-4312-86ed-a1045a314770.png)

Recipe detail page
![CleanShot 2022-07-09 at 08 16 31](https://user-images.githubusercontent.com/1058710/178094250-0e96e18d-4ae4-4ff3-a973-4e3edc5cedef.png)

**Comment by vabene1111** [comment]
can you please veryify this on the latest version (1.3.0) and see if its still happending ?

**Comment by obstschale** [comment]
I just updated to 1.3.0 last night. I can confirm, that the bug still exists.

**Comment by vabene1111** [comment]
thanks, i will look into it

**Comment by vabene1111** [comment]
ok so the problem is that there is a valid url inside the `[]` block that usually contains the link text. After that there is an automatic parser that finds urls that are just put into the text and makes them clickable as well (which is the most convenient way to add links for most people). I am not a parsers expert and dont know how to best fix this (there is likely a possible matching pattern to do so but not sure how currently).

Basically: either just put the link in it will become clickable automatically or put a "non link" text into the text part of the markdown link, then the link works as expected. 

If anyone is good with parsers feel free to take a look at https://github.com/TandoorRecipes/recipes/blob/develop/cookbook/helper/mdx_urlize.py and adapt the patterns to not match links inside markdown link text blocks
